web/src/settings_org.js

```diff
@@ -613,8 +613,10 @@ export function discard_group_property_element_changes(elem, group) {
     const property_name = settings_components.extract_property_name($elem);
     const property_value = settings_components.get_group_property_value(property_name, group);
 
-    if (property_name === "can_manage_group" || property_name === "can_mention_group") {
+    if (property_name === "can_mention_group") {
         settings_components.set_dropdown_list_widget_setting_value(property_name, property_value);
+    } else if (property_name === "can_manage_group") {
+        settings_components.set_group_setting_widget_value(property_name, property_value);
```

Comment:
We'll want to get out of hardcoding lists of elements, and instead doing some item type check. But I guess that'll get easier once we migrate `can_mention_group` too.

Reply:
There is no other setting in the groups UI, so we can also remove the next else-if block now and will not need any if condition once we migrate `can_mention_group`.